ns/rdf-xsd.ttl

```diff
@@ -0,0 +1,283 @@
+  rdfs:label "gMonthDay" ;
+  rdfs:comment "Month and day recurring each year. Lexical form: --MM-DD with optional timezone." .
+
+xsd:duration a rdfs:Datatype ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "duration" ;
+  rdfs:comment "Time durations per ISO 8601 (PnYnMnDTnHnMnS), optionally with a leading '-'. Value space pairs a months component and a seconds component; only a partial order is defined." .
+
+xsd:yearMonthDuration a rdfs:Datatype ;
+  rdfs:subClassOf xsd:duration ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "yearMonthDuration" ;
+  rdfs:comment "Duration restricted to year and month fields (seconds component is 0). Totally ordered; lexical form uses PnYnM with optional leading '-'." .
+
+xsd:dayTimeDuration a rdfs:Datatype ;
+  rdfs:subClassOf xsd:duration ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "dayTimeDuration" ;
+  rdfs:comment "Duration restricted to day, hour, minute, second fields (months component is 0). Totally ordered; lexical form PnDTnHnMnS with optional leading '-'." .
+
+xsd:byte a rdfs:Datatype ;
+  rdfs:subClassOf xsd:short ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "byte" ;
+  rdfs:comment "8-bit signed integer. Value space: [-128 .. 127]." .
+
+xsd:short a rdfs:Datatype ;
+  rdfs:subClassOf xsd:int ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "short" ;
+  rdfs:comment "16-bit signed integer. Value space: [-32768 .. 32767]." .
+
+xsd:int a rdfs:Datatype ;
+  rdfs:subClassOf xsd:long ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "int" ;
+  rdfs:comment "32-bit signed integer. Value space: [-2147483648 .. 2147483647]." .
+
+xsd:long a rdfs:Datatype ;
+  rdfs:subClassOf xsd:integer ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "long" ;
+  rdfs:comment "64-bit signed integer. Value space: [-9223372036854775808 .. 9223372036854775807]." .
+
+xsd:unsignedByte a rdfs:Datatype ;
+  rdfs:subClassOf xsd:unsignedShort ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "unsignedByte" ;
+  rdfs:comment "8-bit unsigned integer. Value space: [0 .. 255]." .
+
+xsd:unsignedShort a rdfs:Datatype ;
+  rdfs:subClassOf xsd:unsignedInt ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "unsignedShort" ;
+  rdfs:comment "16-bit unsigned integer. Value space: [0 .. 65535]." .
+
+xsd:unsignedInt a rdfs:Datatype ;
+  rdfs:subClassOf xsd:unsignedLong ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "unsignedInt" ;
+  rdfs:comment "32-bit unsigned integer. Value space: [0 .. 4294967295]." .
+
+xsd:unsignedLong a rdfs:Datatype ;
+  rdfs:subClassOf xsd:nonNegativeInteger ;
+  rdfs:isDefinedBy <http://www.w3.org/2001/XMLSchema#> ;
+  rdfs:label "unsignedLong" ;
+  rdfs:comment "64-bit unsigned integer. Value space: [0 .. 18446744073709551615]." .
```

Comment:
I think all these ranges should use actual three-dot `...` ellipses instead of the two-dot `..` shortcut (which often reads as a typo or other error).

Reply:
`...` avoids the burden of writing out elements in a list. 

This is not a list, it is a numeric range (inclusive range end uses `[` or`]`, exclusive range end use `(`, `)` - these can be mixed `[0..5)`.  `..` is used for ranges; `,` is also used . `[0, 18446744073709551615]`.